package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@microsoft/logicappsux",
-  "version": "2.0.24",
+  "version": "2.0.25",
```

Comment:
Is this package version update automatic? I tried to recreate the PR without it, but every time I do `git commit -m "..."`, this change gets added.

Reply:
leaving this as is for now, I expect will run into merge conflict at some point, will find out what to do then